pkgs/racket-doc/syntax/scribblings/module-reader.scrbl

```diff
@@ -416,7 +416,9 @@ If loading the module produced by @racket[module-path-parser]
 succeeds, then the loaded module's @racketidfont{read},
 @racketidfont{read-syntax}, or @racketidfont{get-info} export is
 passed to @racket[convert-read], @racket[convert-read-syntax], or
-@racket[convert-get-info], respectively.
+@racket[convert-get-info], respectively. See
+@secref["parse-reader" #:doc '(lib "scribblings/reference/reference.scrbl")]
```

Comment:
You can use `refman` here (as defined in "common.rkt") instead of `'(lib "scribblings/reference/reference.scrbl")`